using-mobile-extensions/adobe-audience-manager/README.md

```diff
@@ -23,33 +23,66 @@ Adobe Audience Manager is a versatile audience data management platform. With th
 2. Import the library.
 
 ```java
-import com.adobe.marketing.mobile.*;
+import com.adobe.marketing.mobile.Audience;
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Identity;
 ```
 
 {% hint style="info" %}
 Audience Manager depends on the Identity extension and is automatically included in the Core pod. When manually installing the Audience Manager extension, ensure that you add the `identity-1.x.x.aar` library to your project.
 {% endhint %}
 {% endtab %}
 
-{% tab title="iOS" %}
-1. Add the library to your project via your `Podfile` by adding `pod 'ACPAudience'`
-2. Import the Audience and Identity library, using the respective language: 
+{% tab title="iOS (AEP 3.x) %}
```

Comment:
Is this missing a terminating `"`?:
`{% tab title="iOS (AEP 3.x) %}` -> `{% tab title="iOS (AEP 3.x)" %}`